static/scss/course-cards.scss

```diff
@@ -46,6 +46,25 @@
           letter-spacing: 0;
           line-height: 19px;
         }
+
+        .course-card-title {
+          color: $primary;
+
+          &:hover {
+            color: #63131f;
+            text-decoration: underline;
+            cursor: pointer;
+          }
+
+        }
+      }
+
+      .course-link {
+        display: block;
+        width: 100%;
+        height: 100%;
+        cursor: inherit;
```

Comment:
Personally I think it makes more sense for the cursor to be a pointer if it's linking to a new page. If you were told that this should just be a normal cursor, or if you have a specific reason for using the default cursor here, you can ignore this suggestion.